contracts/utils/Operators.sol

```diff
@@ -77,13 +74,12 @@ contract Operators is Ownable, IOperators {
     /**
      * @dev Allows an operator to execute arbitrary functions on any smart contract.
      * Can only be called by an operator.
-     * @param target address of the contract to execute the function on
+     * @param target address of the contract to execute the function on. Existence is not checked.
      * @param callData ABI encoded function call to execute on target
      * @return data return data from executed function call
      */
-    function execute(address target, bytes calldata callData) external onlyOperator returns (bytes memory) {
-        // solhint-disable-next-line avoid-low-level-calls
-        (bool success, bytes memory data) = target.call(callData);
+    function execute(address target, bytes calldata callData) external payable onlyOperator returns (bytes memory) {
+        (bool success, bytes memory data) = target.call{ value: msg.value }(callData);
```

Comment:
```suggestion
        (bool success, bytes memory data) = target.call{ value: nativeValue }(callData);
```
for convenience, could also make it so that msg.value is used if nativeValue is `0`. what do you think?

Reply:
If configuring the nativeValue in the function call is optional then yeah, I think we should account for a nativeValue of zero but a nonzero msg.value.